website/content/v1.2/advanced/disaster-recovery.md

```diff
@@ -81,13 +79,20 @@ NODE         NAMESPACE   TYPE          ID             VERSION   TYPE
 172.20.0.3   config      MachineType   machine-type   2         controlplane
 ```
 
-Nodes with `init` type are incompatible with `etcd` recovery procedure.
+Init node type is deprecated, and are incompatible with `etcd` recovery procedure.
 `init` node can be converted to `controlplane` type with `talosctl edit mc --mode=staged` command followed
 by node reboot with `talosctl reboot` command.
 
 ### Preparing Control Plane Nodes
 
 If some control plane nodes experienced hardware failure, replace them with new nodes.
+
+Remove the down nodes *first*, before adding the replacement nodes:
```

Comment:
this is not needed here, as `etcd` is effectively down, and any command like that won't work.

probably the wording above is not clear, but we're talking about the catastrophic failure (when etcd cluster is down)

Reply:
Ah, that does make sense (now) from the context of the headings and subheadings... but wasn't apparent. OK, leave this out, and I'll have a look after the rest of the stuff is committed.